README.md

```diff
@@ -317,13 +318,24 @@ podman run -itv $PWD:/usr/src -v $PWD/cacerts:/opt/java/openjdk/lib/security/cac
 
 ### Seeding data
 
+#### Seeding compliance data
+
 To seed accounts, policies, results and hosts use `dev:db:seed` rake task.
 It is recommeneded to run the command after a first log in was initiated, as it would generate data for that account.
 
 ```
 bundle exec rake dev:db:seed
 ```
 
+#### Seeding RBAC data
+
+```
+# inside of RBAC container
+ACCESS_CACHE_CONNECT_SIGNALS=False MAX_SEED_THREADS=2 ./rbac/manage.py seeds
+```
+
+To provide your own roles, consult https://github.com/RedHatInsights/rbac-config#development.
+
```

Comment:
Would the API work without this? If not, please move it to the one-off container.

Reply:
Without this step, API would be accessible, but `RbacApi.new('...').check_user` would keep returning `false`.

Now that we have `permissions.json` and `definitions.json` copied out from the [rbac-config](https://github.com/RedHatInsights/rbac-config/tree/master/configs/stage) repo into the codebase, this is not necessary and it is being added only for documentation purposes.

Reply:
If this (seeding) step is necessary, please drop it from the docs and move it into the one-off container.

Reply:
The goal is to have an out-of-the box solution for people who don't want to develop RBAC-related stuff, but want to keep using our setup as they had it before your changes, i.e. `podman-compose up -d` should work as before without extra steps.